### ISSUE TITLE
[SPIKE] Add a dummy python server to platform infra

### ISSUE THREAD

**Issue by lmossman** [airbytehq/airbyte#17543]
## Tell us about the problem you're trying to solve
One of the potential concerns called out in [this section](https://docs.google.com/document/d/11HrieUnA7oa6YsDhOVZpAVkOoZt2RWQQWJLPf7xQodY/edit?disco=AAAAhUsdE_A) of the Tech Spec is that the Connector Builder server will be the first airbyte platform deployment that is build on python, so it may have a unique set of challenges that we are unaware of in terms of building and running the python server.

## Describe the solution you’d like
In order to de-risk this part of the project, we should do a spike of adding a dummy python server deployment to the airbyte platform, to uncover any potential issues in that process that we are not aware of.

**This dummy server should be built on the same python server framework we plan to use for the actual implementation** (e.g. Flask or FastAPI, as called out [here](https://docs.google.com/document/d/11HrieUnA7oa6YsDhOVZpAVkOoZt2RWQQWJLPf7xQodY/edit?disco=AAAAhXwfsCI)), to ensure the spike is as accurate as possible. This dummy server can just have a single simple endpoint that returns a successful response.

Once the server is ready, it should be added to the airbyte platform docker deployment and build process. A non-exhaustive list of what may required here:
- A dedicated module in the airbyte repo, with a Dockerfile and build.gradle to define the logic to build a standalone docker image
- Entry added to airbyte/docker-compose.build.yaml so that the container is built properly for docker deployments
- Entry added to airbyte/docker-compose.yaml so that the container is deployed properly for docker deployments
- Updates to airbyte/build.gradle to ensure this image is included in the platform build
- Unique port configured for the container (and a kube NodePort Service with that port for kube deployment if done in this phase)
- A step added to our CI that builds the python server and runs unit tests on it

## Acceptance Criteria
- A dummy python server can be deployed as a standalone deployment in the airbyte platform, which successfully responds to requests coming from other deployments in the platform

**Comment by girarda** [comment]
[FastAPI's tutorial covers deployment to Docker containers](https://fastapi.tiangolo.com/deployment/docker/)

**Comment by girarda** [comment]
grooming notes:
- we've never added a new server to the compose file.
- out of scope: nginx, generating the python code from the openapi spec
- @lmossman to create a follow up issue to configure nginx
- goal: expose a random endpoint
- check with someone on the platform team for CI

**Comment by girarda** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Connector-Operations-6318ede5eda32e0010ef8566/issues/airbytehq/airbyte/17543?planning-poker) with Zenhub @lmossman

**Comment by sherifnada** [comment]
when investigating frameworks, some criteria to keep in mind: 
- how well it handles long running requests
- amount of boilerplate
- handling large requests
- (probably other things there as well)

**Comment by girarda** [comment]
End of sprint update:
* [Link to PoC](https://github.com/airbytehq/airbyte/pull/18410). It's a dummy server, but it goes through the whole CI machinery and is wired with the FE
